{"name": "test_check_email_already_exists_in_change_email", "status": "broken", "statusDetails": {"message": "TypeError: 'TextBox' object is not callable", "trace": "self = <user_settings_tests.TestUserSettings object at 0x10765ba10>\n\n    def test_check_email_already_exists_in_change_email(self):\n        self.garage_page.get_my_profile_button().click()\n        self.garage_page.get_logout_button().click()\n        self.login_page.get_sign_in_button().click()\n        self.register_page.get_registration_button().click()\n        self.fill_all_fields_of_registration_form('Kristina', 'Fedorchuk', 'fedorchuck_kristina@gmail.com',\n                                                  'Kristina1997K', 'Kristina1997K')\n        self.register_page.get_register_button().click()\n        self.settings_page.get_settings_side_menu_button().click()\n>       self.fill_email_and_password_change_email(\"fedorchuck_mykhailo@gmail.com\", \"Kristina1997K\")\n\nuser_settings_tests.py:238: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <user_settings_tests.TestUserSettings object at 0x10765ba10>, email = 'fedorchuck_mykhailo@gmail.com'\npassword = 'Kristina1997K'\n\n    @allure.step(\"Data to fill in email and password fields in 'Change email'\")\n    def fill_email_and_password_change_email(self, email, password):\n>       self.settings_page._new_email_address_field().fill_field(email)\nE       TypeError: 'TextBox' object is not callable\n\nuser_settings_tests.py:211: TypeError"}, "steps": [{"name": "Data to fill fields of registration form", "status": "passed", "parameters": [{"name": "name", "value": "'Kristina'"}, {"name": "last_name", "value": "'Fedorchuk'"}, {"name": "email", "value": "'fedorchuck_kristina@gmail.com'"}, {"name": "password", "value": "'Kristina1997K'"}, {"name": "repeat_password", "value": "'Kristina1997K'"}], "start": 1690334634847, "stop": 1690334635911}, {"name": "Data to fill in email and password fields in 'Change email'", "status": "broken", "statusDetails": {"message": "TypeError: 'TextBox' object is not callable\n", "trace": "  File \"/Users/pauliukevi4i/Stotska_final_project/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/pauliukevi4i/Stotska_final_project/user_settings_tests.py\", line 211, in fill_email_and_password_change_email\n    self.settings_page._new_email_address_field().fill_field(email)\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "email", "value": "'fedorchuck_mykhailo@gmail.com'"}, {"name": "password", "value": "'Kristina1997K'"}], "start": 1690334637578, "stop": 1690334637579}], "start": 1690334633258, "stop": 1690334637579, "uuid": "0f35fddf-d15b-4ccd-a6e1-4c31d676899d", "historyId": "1665b91c622b907a6894bfb50795b4c6", "testCaseId": "1665b91c622b907a6894bfb50795b4c6", "fullName": "user_settings_tests.TestUserSettings#test_check_email_already_exists_in_change_email", "labels": [{"name": "suite", "value": "user_settings_tests"}, {"name": "subSuite", "value": "TestUserSettings"}, {"name": "host", "value": "AirPauliukevich"}, {"name": "thread", "value": "8158-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "user_settings_tests"}]}